v2/passwordless/serverless/with-aws-lambda/setup-lambda.mdx

```diff
@@ -195,5 +153,104 @@ So the value of `apiGatewayPath` should be set according to the above based on t
 You also need to change the `apiBasePath` on the frontend config to append the stage to the path. For example, if the frontend is query the development stage and the value of `apiBasePath` is `/auth`, you should change it to `/dev/auth`.
 :::
 
-<!-- END COPY SECTION -->
+:::note
+You may edit the `apiBasePath` and `apiGatewayPath` value later if you haven't setup the API Gateway yet. 
+:::
+
+
+## 4) Add SuperTokens auth middlewares
+
+Using the editor provided by AWS, create/replace the handler file contents with the following code:
+
+<BackendSDKTabs disableGolang>
+<TabItem value="nodejs">
+
+```tsx title="index.mjs"
+import supertokens from "supertokens-node";
+import { middleware } from "supertokens-node/framework/awsLambda";
+// @ts-ignore
+import { getBackendConfig } from "./config.mjs";
+import middy from "@middy/core";
+import cors from "@middy/http-cors";
+
+supertokens.init(getBackendConfig());
+
+export const handler = middy(
+    // @ts-ignore
+    middleware((event) => {
+      // SuperTokens middleware didn't handle the route, return your custom response
+      return {
+        body: JSON.stringify({
+            msg: "Hello!",
+        }),
+        statusCode: 200,
+      };
```

Comment:
should return a 404 here.

Reply:
Don't worry about the screenshot below changing.. it's ok

Reply:
discussed offline, not changing this.